models/models.go

```diff
@@ -122,3 +122,11 @@ type StepRunResultsModel struct {
 	ErrorStr string                      `json:"error_str" yaml:"error_str"`
 	ExitCode int                         `json:"exit_code" yaml:"exit_code"`
 }
+
+// TestResultStepInfo ...
+type TestResultStepInfo struct {
```

Comment:
may using `stepmanModels.StepInfoModel` would slightly better as it contains every necessary field.

Reply:
I am trying to avoid using that model as the actual code base has this model filled with sensitive data like secrets and I can't just write those out to a file.